server/config.ts

```diff
@@ -48,7 +48,7 @@ const applicationInfo: ApplicationInfo = {
 
 export default {
   applicationInfo,
-  production,
```

Comment:
This is very confusing/misleading, is this needed anywhere?

If it's coming from the TS template maybe we can't remove/rename for some reason.

Reply:
Oh at the very least by `nunjucksSetup.ts` 👌